class EventError(Exception):
    pass


class EventValueError(Exception):
    pass
